INSTALL.md

```diff
@@ -65,6 +68,13 @@ Gentoo
 * gnome-extra/zenity
 * net-misc/curl
 
+Fedora
+=====
+
+```
+sudo cert-sync /etc/pki/tls/certs/ca-bundle.crt
```

Comment:
This should be done in the Fedora **mono-core** RPM package in the `%post` section. If it doesn't, you should report at the RedHat Bugzilla.